libs/portals/admin/signature-collection/src/screens/List/index.tsx

```diff
@@ -47,55 +39,19 @@ const List = () => {
           span={['12/12', '12/12', '12/12', '8/12']}
         >
           {!!list && (
-            <Box>
+            <>
```

Comment:
These empty brackets that include multiple children usually throw the warning 
`Fragments should contain more than one child - otherwise, there’s no need for a Fragment at all.eslint`
should they be <div> instead?

Reply:
my fragments contain more than one child so no warnings ;)